src/consts.ts

```diff
@@ -0,0 +1,7 @@
+export const FIREBASE_API_KEY = process.env.REACT_APP_FIREBASE_API_KEY as string;
```

Comment:
For env variables good to have something like this
export class MissedEnvVarError extends Error {
  constructor(variableName: string) {
    super(`Missed environment variable: ${variableName}`);
    this.name = 'MissedEnvVarError';
  }
}

export const throwMissedEnvVarError = (variableName: string): never => {
  throw new MissedEnvVarError(variableName);
};

 FIREBASE_API_KEY:
        process.env. REACT_APP_FIREBASE_API_KEY ||
        throwMissedEnvVarError('REACT_APP_FIREBASE_API_KEY')